tests/dream/io/geant4_test.py

```diff
@@ -112,7 +112,6 @@ def test_load_geant4_csv_mantle_has_expected_coords(file):
 
     assert "sector" not in mantle.bins.coords
     assert "tof" in mantle.bins.coords
-    assert "wavelength" in mantle.bins.coords
```

Comment:
This will now cause merge conflicts with #71

Reply:
I'll wait for #71 to be merged before rebasing this